enterprise/api/v1/views.py

```diff
@@ -267,15 +375,10 @@ def list(self, request):
         Returns:
             (Response): DRF response object containing course catalogs.
         """
-        # fetch all course catalogs.
```

Comment:
Comments are obviously a good practice but imagine something akin to:

```python
# get the users
users = get_the_users()
```

Such comments can end up being awkward technical debt, interestingly. I deleted these because doing a quick read of the code I felt they didn't contribute much at all to readability -- did you feel differently?